apps/examples/testcase/le_tc/kernel/tc_sched.c

```diff
@@ -524,6 +573,59 @@ static void tc_sched_sched_getstreams(void)
 	TC_SUCCESS_RESULT();
 }
 
+/**
+ * @fn                   :tc_sched_task_setcanceltype
+ * @brief                :This tc tests tc_sched_task_setcanceltype()
+ * @Scenario             :The task_setcanceltype() function atomically both sets the calling
+ *                        task's cancelability type to the indicated type and returns the
+ *                        previous cancelability type at the location referenced by oldtype
+ *                        If successful pthread_setcanceltype() function shall return zero;
+ *                        otherwise, an error number shall be returned to indicate the error.
+ * @API'scovered         :task_setcanceltype
+ * @Preconditions        :none
+ * @Postconditions       :none
+ * @return               :void
+ */
+#ifdef CONFIG_SCHED_WAITPID
+#ifdef CONFIG_CANCELLATION_POINTS
+#ifndef CONFIG_SCHED_CHILD_STATUS
+static void tc_sched_task_setcanceltype(void)
+{
+	pid_t pid;
+	int ret_chk;
+
+	/* creating new process */
+	pid = task_create("sched1", SCHED_PRIORITY_DEFAULT, TASK_STACKSIZE, tc_sched_task_setcanceltype_childthread, (char * const *)NULL);
+	TC_ASSERT_GT("task_create", pid, 0);
+
+	ret_chk = waitpid(pid, &tasksetcanceltype_status, 0);
+
+	/* If TC's pass , only then execute success result */
+	if(tasksetcanceltype_status == 0) {
```

Comment:
Even if tasksetcanceltype_status is set at tc_sched_task_setcanceltype_childthread function, it will be updated by waitpid.
Please refer [[waitpid example]](https://github.com/Samsung/TizenRT/blob/master/apps/examples/kernel_sample/waitpid.c) in kernel_sample. 
```
ret = (int)waitpid(g_waitpids[0], &stat_loc, 0);
WEXITSTATUS(stat_loc) != RETURN_STATUS
```